src/v2/models/AppState.js

```diff
@@ -152,6 +152,16 @@ export default Model.extend({
       reRender = false;
     }
 
+    if (identicalResponse && this.get('currentFormName') === 'poll') {
```

Comment:
nit:
looks like we can clean up the logic a bit.
- `return false` at line 152
- don't need to check `identicalResponse`

Reply:
This wont work since it iwll always return false since identicalResponse is true.